### DISCUSSION TITLE
Language detection is broken on all deepin websites

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#3391] [General | 常规讨论]
Language detection on all deepin websites doesn't work, it defaults to Chinese even if my locales and ip aren't matching. Even after setting English language manually it's being forgotten after going to another page.

**Comment by BLumia** [comment]
Could you press F12 and write the following command in the console, then provide the output?

``` js
console.log(navigator.language)
```

cc @myml

**Reply by raspin0** [reply]
![obraz](https://user-images.githubusercontent.com/39202451/185910343-f779e214-7203-4927-a598-5abf0b9cd79b.png)

**Comment by myml** [comment]
Can you post your browser version and website url?

**Reply by raspin0** [reply]
Browser: Firefox 104
URL: https://deepin.org / https://linglong.dev/

**Reply by myml** [reply]
Thanks for your feedback.

The https://linglong.dev/ site does not support language detection, you can temporarily save  https://linglong.dev/en/index.html to bookmark. @realwujing 

deepin.org should be replaced with www.deepin.org, The new home page does not automatic redirect to www(Fix in next version).

**Reply by myml** [reply]
redirect config completed!
